Makefile

```diff
@@ -1,2 +1,19 @@
-lint:
-	@golangci-lint run --config ./config/golangci.yaml
\ No newline at end of file
+.PHONY: tidy lint test-unit test-main test
+
+tidy:
+	@go mod tidy
+
+lint: tidy
+	@golangci-lint run --config ./config/golangci.yaml
```

Comment:
You have duplicate `golangci.yaml` files: one at the top level of the project and one located in `config/`. Decide which you want to keep and delete the other then update this makefile command accordingly

Reply:
done